src/lib/types.h

```diff
@@ -792,4 +792,13 @@ typedef struct rnp_key_protection_params_t {
     pgp_hash_alg_t    hash_alg;
 } rnp_key_protection_params_t;
 
+/*
+ * Data structure for storing pointer to the buffer together with
+ * it's length. Value of 'len' depends on context in which it is used.
```

Comment:
> Value of 'len' depends on context in which it is used.

That sounds a bit error-prone to me.

Reply:
Indeed, @henrydcase could you fix this a bit and merge? Thanks!

Reply:
I think it is OK as it's just utility.